index.bs

```diff
@@ -77,16 +88,16 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The magnetometer's associated {{Sensor}} subclass is the {{Magnetometer}} class.
-
-The Magnetometer has a <a>default sensor</a>, which is the device's main magnetomer sensor.
+The magnetometer has two associated {{Sensor}} subclasses, {{Magnetometer}} and {{RawMagnetometer}}.
```

Comment:
So the 1:1 relationship between sensor concept and `Sensor` interface isn't explicit in the generic sensor spec, but it's sort of implied (I probably need to clarify that—my bad). How are you planning to handle the shared latest reading map between the two?